docs/index.md

```diff
@@ -21,5 +21,5 @@ There are three main patterns that can be leveraged in the Digital Trust space,
 Depending on which pattern you are looking to implement, the following solutions will provide a foundation to get started:
 
 - [VC-AuthN and SSO](solutions/vc-authn-sso.md): a solution that allows the use of Digital Credentials in an OIDC authentication flow. As an identity Provider service integrated with the [Pathfinder SSO Service](https://developer.gov.bc.ca/docs/default/component/css-docs) it provides a lightweight, standard approach to web application authentication that does not require deep knowledge of Digital trust patterns and tools.
-- [Traction](solutions/traction-overview.md): a Software-As-A-Service Enterprise agent service based on [Aries Cloudagent Python](https://github.com/hyperledger/aries-cloudagent-python), it provides streamlined onboarding for new adopters wanting to have full control over their Digital trust processes, from receiving and presenting Digital credentials to acting as a verifier or even an issuer.
+- [Traction](solutions/traction-overview.md): a Software-As-A-Service Enterprise agent service based on [Aries Cloudagent Python](https://github.com/hyperledger/aries-cloudagent-python), it provides streamlined onboarding for new adopters wanting to have full control over their Digital Trust processes, from receiving and presenting Digital credentials to acting as a verifier or even an issuer.
```

Comment:
There's the hyperledger > OWF renamings as well, not sure if this is the only place but could do those too...

"Aries Cloudagent Python" -> "ACA-Py"

hyperledger -> OWF

Reply:
Oh, good catch! On it.